sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -893,6 +896,12 @@ object SparkSession {
     }
   }
 
+  private def createQueryExecutionListeners(conf: SparkConf): Seq[QueryExecutionListener] = {
+    conf.get(StaticSQLConf.QUERY_EXECUTION_LISTENERS)
+      .map(Utils.classForName(_))
+      .map(_.newInstance().asInstanceOf[QueryExecutionListener])
```

Comment:
Simply throwing `ClassNotFoundException` might not be good to end users, if we plan to make this SQL configuration external. 

Could you use the try and catch to issue a better error message when we are unable to create/initialize the class? Thanks!